R/germline.R

```diff
@@ -74,8 +74,9 @@ germline_single_df <- function(data, species, sample_name = NA) {
     drop_na(Germline.sequence)
 }
 
+# remove everything starting from first ',' or '(' character from the string
```

Comment:
There is a good suggestion regarding comments in tidyverse R code style: "In code, use comments to explain the “why” not the “what” or “how”. "
https://style.tidyverse.org/functions.html#comments-1